ci/requirements-docs.yml

```diff
@@ -10,3 +10,5 @@ pyarrow>=0.12
 sphinx=2.0.1
 sphinx-releases
 sphinx_rtd_theme
+# https://github.com/ibis-project/ibis/issues/2027
```

Comment:
what is pulling this in now?

Reply:
i would create an issue to remove this pin when the underlying issue is resolved.

Reply:
`sphinx-releases` has an issue that a change in `semantic_version` caused. You're right, we should have another issue to remove the pin when it gets fixed.

Reply:
I just created an issue to unpin `semantic_version` once the issue is fixed (#2033)

Reply:
thanks @scottcode ! I adding a release note now here.

Reply:
@xmnlab ok, lgtm. ping when ready.